src/navBar/navbar.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title> Quality Education </title>
+    <link rel="stylesheet" href="NavBar_style.css">
+    <style>
+        head{
+            font-family: Arial, Helvetica, sans-serif;
+        }
+        body{
+            font-family: Arial, Helvetica, sans-serif;
+        }
+
+        footer {
+        background-color: rgb(167, 43, 43);
+        color: white;
+        text-align: center;
+        padding: 20px 0;
+    }
+
+    </style>
+</head>
+<body>
+    <title>Quality Education</title>
+    <nav>        
+    <div class="navbar">
+        <img src="3ed.png" id=logo> 

```

Comment:
i put the img in assets folder, refer from there like this `<img src="../../assets/3ed.png" id=logo> `